action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
     description: Extra contents to write to user .bazelrc
     required: false
     default: ""
+  cache-prefix:
+    description: Key prefix for all caches
```

Comment:
Let's clarify it's only used for disk/repository cache, not bazlisk/external.

Reply:
The way I implemented it, all of the caches use this prefix, including bazelisk and external. This resolves #73